libwallet/assets/dcr/ticket.go

```diff
@@ -204,7 +197,22 @@ func (asset *Asset) VSPTicketInfo(hash string) (*VSPTicketInfo, error) {
 
 	ticketInfo.Client = vspClient
 
-	vspTicketStatus, err := vspClient.GetTicketStatus(ctx, ticketHash)
+	txs, _, err := asset.Internal().DCR.GetTransactionsByHashes(ctx, []*chainhash.Hash{ticketHash})
+	if err != nil {
+		return nil, err
+	}
+
+	commitmentAddr, err := stake.AddrFromSStxPkScrCommitment(txs[0].TxOut[1].PkScript, asset.chainParams)
```

Comment:
Check `len(txs)` first, in case no tx was found for the provided hash.
Also check that `len(txs[0].TxOut) > 1`, because the tx hash may be valid but may not be a ticket purchase tx and may have only 1 output. Actually, is there a way to verify that the tx is a ticket purchase tx before calling the vsp endpoint?

Reply:
I don't think that case scenario will ever happen because we used a valid ticket hash to the transaction.